buildkite.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+cd MUXSDKStats
+pod deintegrate && pod install
+cd ..
+./update-release-frameworks.sh
+zip -r MUXSDKStats.framework.zip Frameworks
+buildkite-agent artifact upload "MUXSDKStats.framework.zip"
```

Comment:
Similar comment to https://github.com/muxinc/mux-stats-sdk-objc/pull/91/files, we should create a `pipeline.yml` config where the build artifacts are specified

Reply:
done here: https://github.com/muxinc/mux-stats-sdk-avplayer/pull/65/commits/508fbf34d1dc4031398cbb14de5e6df12199b609